src/dataviews/histogram-dataview/histogram-data-model.js

```diff
@@ -77,6 +78,10 @@ module.exports = Model.extend({
     this.on('change:column', function () {
       this.set('aggregation', 'auto', { silent: true });
     });
+
+    this.on('sync', function () {
+      this.hasBeenFetched = true;
+    });
```

Comment:
New code and no spec?

![jiji no](https://user-images.githubusercontent.com/1078228/37591064-9092e7a2-2b6a-11e8-87b4-2f573ae46739.gif)

Reply:
Cmon! If I had more time in RT I'd have done it... also, I was just testing if that worked :P